storage/storage.go

```diff
@@ -665,3 +680,11 @@ func startConsumerSpanFromContext(ctx context.Context, operationName string) (ta
 	)
 	return span, ctx
 }
+
+// WithEventhubInfoInBlobPath configures the LeaserCheckpointerOption with a blob path
+func WithEventhubInfoInBlobPath(ehInfo *conn.ParsedConn, consumerGroup string) LeaserCheckpointerOption {
```

Comment:
Actually, on further thought, I dont think this method should be so specific. It should be a more generic options config. Something like:

```
func WithPrefixInBlobPath(prefix string) LeaserCheckpointerOption {
     return func(ls *LeaserCheckpointer) error {
         ls.blobPathPrefix = prefix
         return nil
    }
}
```

Reply:
I agree with your approach, a simple prefix method is good. In this case, I think it may be helpful to include a prefix-building method or a very detailed description in the comment on how to build the prefix with `namespace/suffix/etc.`, just to make people's lives easier.

Which would you prefer ?

Reply:
I'm not sure I'd like to call out the specific checkpointing path that the other language SDKs are using. Particularly, since having both checkpointers in the same location could cause some unknown behavior/conflicts. In my opinion we should leave it generic without specifying a particular path. 
@jhendrixMSFT what do you think?

Reply:
My concern with adding a detailed description to the docs is that people tend to take this as gospel and then blindly use it in their apps.  I don't know if this is good or bad in this case as I don't understand the details, but I would prefer to leave it opaque unless there's a good reason not too.

What happens if SDKs concurrently use the same location?

Reply:
> What happens if SDKs concurrently use the same location?

In the blob path, for checkpoints on the same hub, the distinction is made with the consumer groups. As I see it, I don't think it is a common use-case to have the same consumer group for 2 different applications (written with different SDKs, for instance). Therefore, I don't believe we should be seeing checkpointers in the same location too often.

@catalinaperalta my only concern is that getting eventhub info from `*conn.ParsedConn` is slightly obscure. But I would be happy with simply adding the following in the prefixing method comments (or not adding anything if it's a real concern for you).
```
// "namespace/suffix/hubName/consumerGroup" is one recommended way of suffixing blob paths
```

Reply:
I see the value in what you're saying, but I would rather go out of the door with a simple prefix method and see if others need additional guidance as to what the blob prefix could/should be. Since I'm sure that now that we add this method others will use it for different scenarios. We can always add the comment in later after we get some feedback from other users.